admin/server/server.go

```diff
@@ -394,6 +394,15 @@ func (s *Server) jwtAttributesForUser(ctx context.Context, userID, orgID string,
 		"admin":  projectPermissions.ManageProject,
 	}
 
+	member, err := s.admin.DB.FindOrganizationMemberUsers(ctx, orgID, "", false, user.Email, 1, "")
+	if err != nil {
+		return nil, err
+	}
+
+	if len(member) > 0 && len(member[0].Attributes) > 0 {
+		attr["user"] = member[0].Attributes
+	}
```

Comment:
The `attr` already gets mapped to `.user` in the templating, so this will introduce double nesting. The custom attributes should just be merged into the built-in ones. I think the PRD called for letting custom attributes override the built-in ones, so this should do it:
```suggestion
	if len(member) > 0 {
		for k, v := range member[0].Attributes {
			attr[k] = v
		}
	}
```